recipes/themysto/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# this is a conda-forge recipe. See https://github.com/conda-forge for details
+
+{% set version = "0.0.4" %}
+
+package:
+  name: themysto
+  version: {{ version }}
+
+source:
+  # also taken from nb_conda
+  git_url: https://github.com/jcb91/IPython-notebook-extensions.git
+  git_tag: conda
```

Comment:
Also, is it alright if we use the original repo to get these ( https://github.com/ipython-contrib/IPython-notebook-extensions )?